inference-engine/thirdparty/clDNN/kernel_selector/core/cl_kernels/lrn_gpu_across_channel_multiple_features_fsv16.cl

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) 2020 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "include/common.cl"
+#include "include/data_types.cl"
+#include "include/fetch.cl"
+
+__attribute__((intel_reqd_sub_group_size(16)))
+KERNEL (lrn_gpu_across_channel_multiple_features_fsv16)(
+    const __global INPUT0_TYPE* input,
+    __global OUTPUT_TYPE* output
+#if HAS_FUSED_OPS_DECLS
+    , FUSED_OPS_DECLS
+#endif
+    )
+{
+    const uint feature_id   = (uint)get_global_id(0);
+    const uint x            = (uint)get_global_id(1);
+    const uint b_y          = (uint)get_global_id(2);
+    const uint batch_id     = b_y / INPUT0_SIZE_Y;
+    const uint y            = b_y % INPUT0_SIZE_Y;
+
+    if (feature_id >= INPUT0_FEATURE_NUM)
+        return;
+
+    int input_offset_f = feature_id - PADDING;
+
+    float val[LOCAL_SIZE];
+    float res = 0;
```

Comment:
Because there are different `INPUT0_TYPE` possible (such as `INT8` etc). I would like to keep necessary accuracy in calculations and this is why I made all computations in float and convert it to `INPUT0_TYPE` only before saving results.